Formula/jumanpp.rb

```diff
@@ -0,0 +1,24 @@
+class Jumanpp < Formula
+  desc "Japanese Morphological Analyzer based on RNNLM"
+  homepage "http://nlp.ist.i.kyoto-u.ac.jp/EN/index.php?JUMAN%2B%2B"
+  url "http://lotus.kuee.kyoto-u.ac.jp/nl-resource/jumanpp/jumanpp-1.01.tar.xz"
+  sha256 "0d587416a3eb7123638f9c1e30a649b72dfb483448839168dcb48be572c5919a"
+
+  depends_on "boost-build" => :build
+  depends_on "gperftools" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # Currently there is a bug in Makefile.
+    # `echo "" | src/jumanpp -D juman-resources` is too slow so that make process exits with error code.
+    # Preventing this bug, disable parallel build
+    system "make", "-j1"
```

Comment:
Can you please submit this issue to the upstream developers of this project and add a link to the upstream submission and explanation of why `ENV.deparallelize` (instead of `-j1`) is needed in a comment in the formula file. Thanks!